config/nginx.conf

```diff
@@ -0,0 +1,31 @@
+# This is the version used in development environments
+server {
+    listen 8080;
```

Comment:
Is it possible to change this to `8079` and then change the port for `uwsgi` to something else? It would be nice to keep the same port number so all the passwords and history in our browsers can stay the same